<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrogramControls</class>
 <widget class="QWidget" name="SpectrogramControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>106</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sample Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_wordSize">
       <property name="suffix">
        <string> bit</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_wordFormat"/>
     </item>
     <item>
      <widget class="QComboBox" name="cb_endianness"/>
     </item>
     <item>
      <widget class="QComboBox" name="cb_dataType"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sample Rate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sb_sampleRate">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
       <property name="value">
        <double>8000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_rateUnits"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lb_fftSize">
       <property name="text">
        <string>FFT Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_fftSize">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>32</number>
       </property>
       <property name="maximum">
        <number>131072</number>
       </property>
       <property name="value">
        <number>2048</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Overlap:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_overlap">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sensitivity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hs_sensitivity">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="sliderPosition">
        <number>2500</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_logarithmic">
       <property name="text">
        <string>Logarithmic</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_showHeaders">
       <property name="text">
        <string>Show Headers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_hoverSlices">
       <property name="text">
        <string>Show Hover Slices</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
